e2e_tests/copy_controller.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { test, expect } from "@playwright/test"
+
+test.describe("📋 Copy Controller", () => {
```

Comment:
quid de nommer ce test avec la fonctionnalité testé


```suggestion
test.describe("📋 Test de la fonctionnalité de partage > copie d'une fiche acteur", () => {
```